src/Giraffe/HttpHandlers.fs

```diff
@@ -336,6 +336,19 @@ let xml (dataObj : obj) : HttpHandler =
     setHttpHeader "Content-Type" "application/xml"
     >=> setBody (serializeXml dataObj)
 
+/// Returns the stream as PDF to the client.
+let pdf fileName (stream : Stream) : HttpHandler =
+    fun (next : HttpFunc) (ctx : Http.HttpContext) ->
+        task {
+            let l = int stream.Length
+            stream.Position <- int64 0
```

Comment:
usually that's done before. because not all stream support seeking.
functions shouldnt care about current position